### DISCUSSION TITLE
Transfer des donnees sur autre telephone

### DISCUSSION THREAD

**Discussion by pipodug** [osmandapp/OsmAnd#15686] [Q&A]
Bonjour
Je viens de changer de téléphone et j'ai bien transférer l'app osmand sur le nouveau mais les données n'ont pas été transférées, surtout les traces qui ont été importées.
Y a-t-il un moyen de récupérer les données de l'app de l'ancien téléphone pour les mettre sur le nouveau ?
C'est surtout pour les traces importées et enregistrées. 
Merci pour toute aide !

**Comment by mariush444** [comment]
Main menu -> settings -> backup & restore
https://maptile.osmand.net/docs/user/personal/storage#backup--restore
